stock_available_mrp/models/product_template.py

```diff
@@ -1,46 +1,20 @@
 # -*- coding: utf-8 -*-
-# © 2014 Numérigraphe SARL
+# Copyright 2014 Numérigraphe SARL
+# Copyright 2016-17 Sodexis, Inc. <dev@sodexis.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields, api
-from openerp.addons import decimal_precision as dp
+from odoo import models, api
 
 
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
-    potential_qty = fields.Float(
-        compute='_get_potential_qty',
-        type='float',
-        digits_compute=dp.get_precision('Product Unit of Measure'),
-        string='Potential',
-        help="Quantity of this Product that could be produced using "
-             "the materials already at hand. "
-             "If the product has several variants, this will be the biggest "
-             "quantity that can be made for a any single variant.")
-
     @api.multi
     @api.depends('potential_qty')
-    def _immediately_usable_qty(self):
+    def _compute_immediately_usable_qty(self):
         """Add the potential quantity to the quantity available to promise.
 
         This is the same implementation as for variants."""
-        super(ProductTemplate, self)._immediately_usable_qty()
+        super(ProductTemplate, self)._compute_immediately_usable_qty()
         for tmpl in self:
             tmpl.immediately_usable_qty += tmpl.potential_qty
```

Comment:
See pr #249. There is a performance issue computing quantities using the product field (potential qty) like this.

Prefer compute quantity with modifying dictionary returned by product_available method.